### PR TITLE
chore(django) Upgrade to Django1.8

### PR DESCRIPTION
**Pull request by markstory**
Updating requirements to pin to django 1.8

Refs SEN-546

### CODE REVIEW

**Comment by wedamija** [comment]
Didn't we still need to convert our migrations over to the new Django migration format?

**Comment by markstory** [comment]
@wedamija Not right away. South still works in django18. Cramer has another pull request that moves from south to django_migrations. I am planning on getting those merged once django18 lands.

File: .travis.yml [hunk 1]
```diff
@@ -130,22 +130,6 @@ matrix:
...
 
-    # django 1.8 compatibility
-    - <<: *postgres_default
-      name: 'Backend [Postgres] (Django 1.8) (1/2)'
-      env: TEST_SUITE=postgres DB=postgres DJANGO_VERSION=">=1.8,<1.9" TOTAL_TEST_GROUPS=2 TEST_GROUP=0
-    - <<: *postgres_default
-      name: 'Backend [Postgres] (Django 1.8) (2/2)'
-      env: TEST_SUITE=postgres DB=postgres DJANGO_VERSION=">=1.8,<1.9" TOTAL_TEST_GROUPS=2 TEST_GROUP=1
-
-    # django 1.8 compatibility with migrations
```
Comment (wedamija) [inline .travis.yml]: Do we need to keep this `SOUTH_TESTS_MIGRATE=0` test in place?

**Review by wedamija — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (markstory) [reply .travis.yml]: I can switch up the tests to run without migrations as with them they are a fair bit slower.

**Pull request merged**